cronjobs/forwarder/index.ts

```diff
@@ -217,7 +217,7 @@ const handleReconciliation = async (uniqueExternalPaymentIds: string[]) => {
       status: status as PaymentStatus,
       preimage: preimage,
       msatAmount: parseInt(amount),
-      fee: parseInt(fee),
+      fee: isNaN(parseInt(fee)) ? 0 : parseInt(fee),
```

Comment:
something like `parseInt(fee) || 0` should also work, since `NaN` is falsey
```suggestion
      fee: parseInt(fee) || 0,
```